app/uk/gov/hmrc/helloworldupscan/config/ErrorHandler.scala

```diff
@@ -25,7 +25,7 @@ import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
 import uk.gov.hmrc.helloworldupscan.views
 
 @Singleton
-class ErrorHandler @Inject()(val messagesApi: MessagesApi, implicit val appConfig: AppConfig) extends FrontendErrorHandler {
+class ErrorHandler @Inject()(errorView: views.html.error_template)(val messagesApi: MessagesApi, implicit val appConfig: AppConfig) extends FrontendErrorHandler {
```

Comment:
Can we rename the views to have Scala standard types? `ErrorTemplate` and `GovUkWrapper`